### DISCUSSION TITLE
3.36 Error

### DISCUSSION THREAD

**Discussion by Noreas** [djravine/idlecombos#25] [Q&A]
![изображение](https://user-images.githubusercontent.com/23726199/212091447-206d6b52-2fa1-42fe-824d-abc36bea0034.png)

**Comment by djravine** [comment]
What version of AHK are you using?
I have the latest [v1.1.36.0](https://www.autohotkey.com/download/ahk-install.exe) and cannot seem to reproduce this error.

**Reply by djravine** [reply]
I have found a potential fix using `switch true {...}`

**Comment by djravine** [comment]
Issue https://github.com/djravine/idlecombos/issues/26

**Reply by djravine** [reply]
New release [v3.37](https://github.com/djravine/idlecombos/releases/tag/v3.37) to see if this fixes your `switch` error.

**Comment by djravine** [comment]
Please let me know if this fixes your error.

**Reply by djravine** [reply]
I'm assuming the issue is fixed.. locking discussion
